package.json

```diff
@@ -1,94 +1,94 @@
 {
-  "name": "@cheapreats/react-ui",
-  "version": "2.3.17",
-  "description": "React UI library for CheaprEats",
-  "main": "dist/index.js",
-  "files": [
-    "dist"
-  ],
-  "sideEffects": false,
-  "scripts": {
-    "start": "start-storybook -p 6006",
-    "deploy": "build-storybook --quiet -o .deploy",
-    "build": "rm -rf dist && tsc && NODE_ENV=production babel dist -d dist",
-    "test": "tsc",
-    "prepublish": "npm run lint && npm run build",
-    "eslint": "eslint ./src --ext tsx,ts,jsx,js",
-    "eslint:fix": "npm run eslint -- --fix",
-    "prettier": "prettier '**/*.ts' '**/*.tsx' '**/*.js' '**/*.jsx'",
-    "prettier:fix": "npm run prettier -- --write",
-    "lint": "npm run eslint:fix && npm run prettier:fix"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/CheaprEats/react-ui.git"
-  },
-  "author": "CheaprEats Inc.",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/CheaprEats/react-ui/issues"
-  },
-  "homepage": "https://github.com/CheaprEats/react-ui#readme",
-  "peerDependencies": {
-    "react": "*",
-    "react-dom": "*",
-    "styled-components": "*",
-    "styled-icons": "*"
-  },
-  "devDependencies": {
-    "@babel/cli": "^7.8.3",
-    "@babel/core": "^7.8.3",
-    "@babel/preset-env": "^7.8.3",
-    "@babel/preset-react": "^7.8.3",
-    "@storybook/addon-actions": "^5.3.9",
-    "@storybook/addon-info": "^5.3.9",
-    "@storybook/addon-knobs": "^5.3.9",
-    "@storybook/addon-links": "^5.3.9",
-    "@storybook/addon-notes": "^5.3.9",
-    "@storybook/addons": "^5.3.9",
-    "@storybook/react": "^5.3.9",
-    "@types/react-dom": "^16.9.5",
-    "@types/storybook__react": "^4.0.2",
-    "@types/styled-components": "^4.4.2",
-    "@typescript-eslint/eslint-plugin": "^1.13.0",
-    "@typescript-eslint/parser": "^1.13.0",
-    "awesome-typescript-loader": "^5.2.1",
-    "babel-loader": "^8.0.6",
-    "babel-plugin-module-resolver": "^3.2.0",
-    "eslint": "^5.16.0",
-    "eslint-config-airbnb": "^17.1.1",
-    "eslint-config-prettier": "^6.9.0",
-    "eslint-import-resolver-alias": "^1.1.2",
-    "eslint-plugin-import": "^2.20.0",
-    "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-react": "^7.18.0",
-    "husky": "^3.1.0",
-    "prettier": "^1.19.1",
-    "react": "^16.12.0",
-    "react-docgen-typescript-loader": "^3.6.0",
-    "react-dom": "^16.12.0",
-    "styled-components": "^5.0.0",
-    "styled-icons": "^7.15.1",
-    "typescript": "^3.7.5"
-  },
-  "aliases": {
-    "@Containers": "src/Containers",
-    "@Layouts": "src/Fragments",
-    "@Inputs": "src/Inputs",
-    "@Text": "src/Text",
-    "@Themes": "src/Themes",
-    "@Utils": "src/Utils"
-  },
-  "eslintIgnore": [
-    "stories",
-    "scripts",
-    "node_modules",
-    "dist",
-    ".storybook",
-    ".deploy"
-  ],
-  "dependencies": {
-    "@types/react-image-crop": "^8.1.2",
-    "react-image-crop": "^8.5.0"
-  }
+    "name": "@cheapreats/react-ui",
+    "version": "2.3.20",
```

Comment:
Why did you use 20 instead of 18?

Reply:
On npm it said .19
![image](https://user-images.githubusercontent.com/33525942/73770458-77b93580-474a-11ea-81d8-794bbc14a1f2.png)

Reply:
I figured ok it’s good then to merge!